packages/d3fc-series/src/canvas/multi.js

```diff
@@ -18,15 +18,19 @@ export default () => {
                 .xScale(xScale)
                 .yScale(yScale);
 
-            const adaptedDecorate = dataSeries.decorate();
-            dataSeries.decorate((c, d, i) => {
-                base.decorate()(c, data, index);
-                adaptedDecorate(c, d, i);
-            });
-
-            dataSeries(seriesData);
-
-            dataSeries.decorate(adaptedDecorate);
+            if (dataSeries.decorate) {
+                const adaptedDecorate = dataSeries.decorate();
+                dataSeries.decorate((c, d, i) => {
+                    base.decorate()(c, data, index);
+                    adaptedDecorate(c, d, i);
+                });
+
+                dataSeries(seriesData);
+
+                dataSeries.decorate(adaptedDecorate);
+            } else {
+                dataSeries(seriesData);
```

Comment:
Minor stylistic nit pic - I'm not a fan of duplicating this line.

Reply:
I agree with this point but I wasn't sure how to refactor. I'll take another look.